tests/integration/targets/elb_classic_lb/tasks/simple_instances.yml

```diff
@@ -18,6 +18,7 @@
 - name: Wait for instance a
   ec2_instance:
     name: "ansible-test-{{ tiny_prefix }}-elb-a"
+    image_id: "{{ images_info.images | sort(attribute='creation_date') | reverse | first | json_query('image_id') }}"
```

Comment:
These instances should already exist (setup_instances.yml) Might be better to use the instance IDs (`{{ instance_a }}` and `{{ instance_b }}`) rather than adding the image_id in here.